### DISCUSSION TITLE
Create a high level alert in office365

### DISCUSSION THREAD

**Discussion by Al19172** [wazuh/wazuh#13537] [Q&A] [answered]
Hello,
How i can create a validation that when user fails to log in multiple times in office365, wazuh send a high level alert?

**Comment by mikykeane** [comment, marked as answer]
Hello @Al19172 , 

in order to create such an alert, you can follow the same approach that we have on our brute force detection rules. For this, you will use our frequency option. More detailed information [here.](https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/rules.html#rule) 

First of all, what is the rule ID that you receive when a user fails to log in?  I will assume it's "100100" for the following example. You will have to make a child rule that looks similar to this: 

```
<rule id="100105" level="10" frequency="8" timeframe="120">
  <if_matched_sid>100100</if_matched_sid>
  <same_srcuser />
  <description>Multiple failed logins in Office 365 from the user:  $(srcuser)  </description>
</rule>
``` 

In this example, the rule ID 100105 with level 10, will trigger when there are 8 failed logins in the last 120 seconds with the same `srcuser` value. If you are using different fields, make sure to look at all our options [here](https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/rules.html#overview), specifically those that start with `same_<field>`, as you can use any of these on your custom rules. 

 Let me know if any of this was helpful. I will be happy to guide you further. If you have trouble with this, send me the rules that you are working on and I will review them to ensure they work as expected!

Best regards, 
Miguel Keane

**Reply by Al19172** [reply]
Hello.
I got to know more about it but unfortunately it didn't work
I am trying to use the rule id 91545 (Office 365: Secure Token Service (STS) logon events in Azure Active Directory - 0755-office365_rules.xml) and when the user fails the login 5 times, a level 8 alert is created
The first file attached is the rule that i what to see
![Capturar](https://user-images.githubusercontent.com/105227404/169810636-ebd4b408-6f91-4dba-b7b8-d5f826ba3e1a.PNG)
and the second one is the rule that you tell (i put in the same file 0755-0ff9ce365_rules.xml)
![Capturar](https://user-images.githubusercontent.com/105227404/169810909-c8ff864a-bc19-47c5-9820-b8a0924ade71.PNG)
